release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -71,9 +71,34 @@ This release adds improvements related to the following components and concepts:
 [id="ocp-release-notes-ibm-z_{context}"]
 === {ibm-z-title} and {ibm-linuxone-title}
 
+With this release, {ibm-z-name} and {ibm-linuxone-name} are now compatible with {product-title} {product-version}. You can perform the installation with z/VM, LPAR, or {op-system-base-full} Kernel-based Virtual Machine (KVM). For installation instructions, see
+xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Preparing to install on {ibm-z-title} and {ibm-linuxone-title}].
+
+[IMPORTANT]
+====
+Compute nodes must run {op-system-first}.
+====
+
+[discrete]
+[id="ocp-4-18-ibm-z-enhancements_{context}"]
+==== {ibm-z-title} and {ibm-linuxone-title} notable enhancements
+
+The {ibm-z-name} and {ibm-linuxone-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.
+
+This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:
+
+* Adding compute nodes to on-premise clusters using `oc`
```

Comment:
```suggestion
* Adding compute nodes to on-premise clusters using {oc-first}
```
This looks like the first mention of the OpenShift CLI in the release notes as far as I can see by searching.